content/post/prioritization_meeting.md

```diff
@@ -0,0 +1,60 @@
+Others are welcome to sit in or dial in, but at minimum some stakeholder representation is required.
+
+The purpose of the meeting is essentially for stakeholders to work with Data to come up with a rank-ordered list of tickets for consideration in Data's next sprint. The entire group must come to a consensus around which ticket should be ranked first, second, etc... The meeting facilitator literally moves tickets up and down the board and the group decides (debates?) where a ticket should land. This way the Data team's workload is aligned with the organizations priorities without Data having to chase down stakeholders and decide what is the most important intitiative this sprint. If Marketing and Product disagree on what is most important, the entire room together can weigh in and in the end one of the tickets is ranked above the other. When prioritization works well, it is a thing of beauty for some of the following reasons.
+
+### Setting stakeholder expectations
+
+As mentioned, with everyone in the room representatives of each function that attend the meeting get insight into what the Data team is working on. As a result stakeholders are less likely to be confused as to why their work is not being prioritized. This reduces possible resentment or the potential for slacking analysts with extraneous requests. The prioritization meeting establishes a two-way communication between Data and the rest of the org. We acknowledge that we understand business priorities and will put a good faith effort to working on it. And if we don't include a ticket in the sprint, the onus is on us to explain why. We are all held accountable.
+
+### Workload transparency
+
+If Marketing concedes that Product's A/B test analysis is the most important thing Data can work on this sprint, then when they ask why we don't have time to work on their attribution model and we point to the A/B test, the conversation is much easier since there is a built in understanding. Better yet, Marketing can look on our board (assuming project management is visible to the org) and see what we are working on. This transparency means less chance for misunderstanding around Data team priorities.
+
+### Better quality tickets
+
+Due to the consensus-based approach and transparent nature of the prioritization meeting, each function must persuade every other function that their ticket is most important, or at least more important than another team's. This organically biases each request to be well thought out and clearly depicted because when Biz Dev needs to convince a room full of non-Biz Dev folks that their research project should be prioritized, it is in their best interest to have more than a one line ticket description. Usually the more well formulated the
```

Comment:
I think the first sentence works better without the begining clause.
drop:
>Due to the consensus-based approach and transparent nature of the prioritization meeting
and see what that looks like.

Then it begins stronger, "Each function _must_ "